octobot_trading/data/order.py

```diff
@@ -100,10 +100,6 @@ def update(self, symbol, order_id="", status=OrderStatus.OPEN,
             self.origin_price = price
             changed = True
 
-        # TODO
-        # if fee is not None and self.fee != fee and self.filled_quantity is not None:
-        #     self.fee = self.get_computed_fee()
-
```

Comment:
Will be updated in https://github.com/Drakkar-Software/OctoBot-Trading/pull/110/files
into: 
```python
        if fee is not None and self.fee != fee:
            self.fee = fee
```
(fee is used in parsed historical trades)